bbb-common-message/src/main/scala/org/bigbluebutton/common2/msgs/CaptionMessages.scala

```diff
@@ -9,6 +9,22 @@ object AddCaptionLocalePubMsg { val NAME = "AddCaptionLocalePubMsg" }
 case class AddCaptionLocalePubMsg(header: BbbClientMsgHeader, body: AddCaptionLocalePubMsgBody) extends StandardMsg
 case class AddCaptionLocalePubMsgBody(locale: String)
 
+object CaptionSubmitTranscriptPubMsg { val NAME = "CaptionSubmitTranscriptPubMsg" }
+case class CaptionSubmitTranscriptPubMsg(header: BbbClientMsgHeader, body: CaptionSubmitTranscriptPubMsgBody) extends StandardMsg
+case class CaptionSubmitTranscriptPubMsgBody(
+                                              transcriptId: String,
+                                              transcript:   String,
+                                              locale:       String,
+                                              captionType:  String,
+                                            )
+object CaptionSubmitTranscriptSuccessEvtMsg { val NAME = "CaptionSubmitTranscriptSuccessEvtMsg" }

```

Comment:
I suggest naming it simply `CaptionSubmitTranscriptEvtMsg`. The term "EvtMsg" already implies the confirmation of the action, making it unnecessary to include the word "Success".